src/site/pages/digital/src/utils/CircuitInfo/DigitalCircuitInfoHelpers.ts

```diff
@@ -134,7 +134,7 @@ export function GetDigitalCircuitInfoHelpers(store: AppStore, canvas: RefObject<
             const circuitCopyMetadata = await CreateUserCircuit(user.auth, circuitCopy);
 
             // Load circuit copy onto canvas
-            await helpers.LoadCircuit(() => LoadUserCircuit(user.auth, circuitCopyMetadata.getId()));
+            await helpers.LoadCircuit(() => LoadUserCircuit(user.auth, circuitCopyMetadata!.getId()));
```

Comment:
Probably better to have an error check here, what if `CreateUserCircuit` actually returns undefined